go-client/client/client.go

```diff
@@ -4,7 +4,7 @@
 // To get started, use client.NewClient to connect to the server. The Client can then be used to perform operations.
 // See the provided examples in the examples/ folder or the individual code documentation for more.
 //
-// Online docs for the client can be found at /* TODO: deephaven.io link goes here once docs are hosted. */
+// Online docs for the client can be found at https://pkg.go.dev/github.com/deephaven/deephaven-core/go/client
```

Comment:
Just a bit of an ourobouros problem going on. This link will show up in the package docs, so I have to put it there before we create the docs, but the link won't work until we create the docs. Hopefully I got the URL right so it will work once we publish the first version.